src/libs/splashscreen/splashscreen.tsx

```diff
@@ -1,8 +1,10 @@
 import React, { createContext, useCallback, useContext, memo, useState, useMemo } from 'react'
-import SplashScreen from 'react-native-splash-screen'
+import SplashScreen from 'react-native-lottie-splash-screen'
 
 import { SplashScreenContextInterface } from './types'
 
+const MIN_SPLASHSCREEN_DURATION = 2000 // in milliseconds
```

Comment:
je suggère de renommer la constante pour rendre explicite l'unité plutot que de mettre un commentaire

```suggestion
const MIN_SPLASHSCREEN_DURATION_IN_MS = 2000
```